google-cloud-spanner/lib/google/cloud/spanner/convert.rb

```diff
@@ -211,6 +211,20 @@ def grpc_value_to_object value, type
               else
                 value.number_value
               end
+            when :FLOAT32
+              if value.kind == :string_value
```

Comment:
seems this block is repeated for float 32 and 64 can we pull it out ?

Reply:
You're right. I missed this. Fixed it now.